pkg/mapper/configmap/mapper.go

```diff
@@ -11,6 +12,27 @@ type ConfigMapMapper struct {
 	*MapStore
 }
 
+const (
+	metricNS = "aws_iam_authenticator"
+)
+
+// metrics are handles to the collectors for prometheus for the various metrics we are tracking.
+type metrics struct {
+	watch *prometheus.GaugeVec
+}
+
+func createMetrics() metrics {
+	m := metrics{
+		watch: prometheus.NewGaugeVec(prometheus.GaugeOpts{
+			Namespace: metricNS,
+			Name:      "authenticator_watch_success",
```

Comment:
would it make more sense to call it `authenticator_watch_active` (since there is no related failure gauge) ?

Reply:
can the name be `apiServerWatch` ..`watch` is very generic